lib/AppInfo/Application.php

```diff
@@ -119,6 +119,7 @@ public function registerScripts() {
 				function () {
 					\OCP\Util::addScript('richdocuments', 'viewer/viewer');
 					\OCP\Util::addStyle('richdocuments', 'viewer/odfviewer');
+					\OCP\Util::addScript('richdocuments', 'viewer/extrapermissions');
```

Comment:
only add if server supports it? add a check based on the server version maybe ....

Reply:
@DeepDiver1975 do you have an example of app where such check is done? I cannot find any standard way of how to check server version

Reply:
I also need to find a way of checking for version for "$share->getExtraSharePermissions".